pyomo/repn/tests/baron/test_baron.py

```diff
@@ -137,7 +137,7 @@ def __init__(self, *args, **kwds):
 
 
 #class TestBaron_writer(unittest.TestCase):
-class TestBaron_writer(object):
+class XTestBaron_writer(object):
```

Comment:
Note that I didn't deactivate tests.  This class does not inherit from unittest, so these tests should never be run.  But recent versions of nose somehow find and try to run these tests, and hence errors are generated.  This edit merely tried to avoid that issue.

Whether or not you think these tests *should* be run is a separate issue.  I'm not sure about that.